libraries/api/discussion_helper.cpp

```diff
@@ -247,7 +247,12 @@ namespace golos { namespace api {
 
             uint128_t reward_tokens = uint128_t(payout);
 
-            share_type curation_tokens = ((reward_tokens * db.get_curation_rewards_percent())
+            auto curation_rewards_percent = db.get_curation_rewards_percent();
+            if (db.has_hardfork(STEEMIT_HARDFORK_0_19__324)) {
+                curation_rewards_percent = d.curation_rewards_percent;
+            }
```

Comment:
Move logic with `db.has_hardfork()` into `db.get_curation_rewards_percent()`

Refactor `get_curation_rewards_percent()` to accept the comment as parameter.

Reply:
@afalaleev I thought about it, but don't made it such way, because `get_curation_rewards_percent(comment)` was looking unlogic for me.
I was wrong, actually developer could add any extra parameters to any existing function if this function isn't for RPC. Yes?

Reply:
Yes, you may refactor existing functions.
In case of `get_curation_rewards_percent` it will allow to minimize lines of code.